ci/tlk.ps1

```diff
@@ -385,43 +385,54 @@ class TlkRecipe
+        }
+
+        & 'MSBuild.exe' "$(Join-Path $ActionsProjectPath 'DevolutionsGateway.Installer.Actions.sln')" "/p:Configuration=$TargetConfiguration" "/p:Platform=$TargetArch" | Out-Host
+
+        $HeatArgs = @('dir', "$DGatewayPSModulePath",
+            '-dr', 'D.DGATEWAYPSROOTDIRECTORY',
+            '-cg', 'CG.DGatewayPSComponentGroup',
+            '-var', 'var.DGatewayPSSourceDir',
+            '-nologo', '-srd', '-suid', '-scom', '-sreg', '-sfrag', '-gg')
+        
+        & 'heat.exe' $HeatArgs + @('-t', 'HeatTransform64.xslt', '-o', "$($this.PackageName)-$TargetArch.wxs") | Out-Host
+
         $WixExtensions = @('WixUtilExtension', 'WixUIExtension', 'WixFirewallExtension')
         $WixExtensions += $(Join-Path $(Get-Location) 'WixUserPrivilegesExtension.dll')
         
         $WixArgs = @($WixExtensions | ForEach-Object { @('-ext', $_) }) + @(
             "-dDGatewayPSSourceDir=$DGatewayPSModulePath",
             "-dDGatewayExecutable=$DGatewayExecutable",
-            "-dVersion=$ShortVersion", "-v")
-        
-        $WixFiles = @('DevolutionsGateway', "DevolutionsGateway-$TargetArch", "WixUI_CustomInstallDir")
+            "-dVersion=$ShortVersion",
+            "-dActionsLib=$(Join-Path $ActionsProjectPath $TargetArch $TargetConfiguration 'DevolutionsGateway.Installer.Actions.dll')",
+            "-v")
         
-        $HeatArgs = @('dir', "$DGatewayPSModulePath",
-            "-dr", "DGATEWAYPSROOTDIRECTORY",
-            "-cg", "DGatewayPSComponentGroup",
-            '-var', 'var.DGatewayPSSourceDir',
-            '-nologo', '-srd', '-suid', '-scom', '-sreg', '-sfrag', '-gg')
-        
-        & 'heat.exe' $HeatArgs + @('-t', 'HeatTransform64.xslt', '-o', "$($this.PackageName)-$TargetArch.wxs") | Out-Host
-        
-        $InputFiles = $WixFiles | ForEach-Object { "$_.wxs" }
-        $ObjectFiles = $WixFiles | ForEach-Object { "$_.wixobj" }
+        $WixFiles = Get-ChildItem -Include '*.wxs' -Recurse 
+
+        $InputFiles = $WixFiles | Foreach-Object { Resolve-Path $_.FullName -Relative }
+        $ObjectFiles = $WixFiles | ForEach-Object { $_.BaseName + '.wixobj' }
 
         $Cultures = @('en-US', 'fr-FR')
         
         foreach ($Culture in $Cultures) {
-            & 'candle.exe' "-nologo" $InputFiles $WixArgs "-dPlatform=$TargetArch" | Out-Host
+            & 'candle.exe' '-nologo' $InputFiles $WixArgs "-dPlatform=$TargetArch" | Out-Host
             $OutputFile = "$($this.PackageName)_${Culture}.msi"
         
             if ($Culture -eq 'en-US') {
                 $OutputFile = "$($this.PackageName).msi"
             }
         
-            & 'light.exe' "-v" "-nologo" $ObjectFiles "-cultures:${Culture}" "-loc" "$($this.PackageName)_${Culture}.wxl" `
+            & 'light.exe' "-nologo" $ObjectFiles "-cultures:${Culture}" "-loc" "$($this.PackageName)_${Culture}.wxl" `
                 "-out" $OutputFile $WixArgs "-dPlatform=$TargetArch" "-sice:ICE61" | Out-Host
```

Comment:
You changed `"-nologo"` for `'-nologo'` at one place and not the other. I don’t think this is a big deal but just in case there is a deeper reason I’m unaware of

Reply:
No deeper reason. I iterated this file a bit, and that was probably a change that came in via my editor and didn't really notice.